openjdk/allsources.lst

```diff
@@ -10250,7 +10250,7 @@ sun/security/jgss/wrapper/SunNativeProvider.java
 @OPENJDK@/jdk/src/share/classes/javax/swing/plaf/basic/BasicComboBoxEditor.java
 @OPENJDK@/jdk/src/share/classes/javax/swing/plaf/basic/BasicComboBoxRenderer.java
 @OPENJDK@/jdk/src/share/classes/javax/swing/plaf/basic/BasicComboBoxUI.java
-@OPENJDK@/jdk/src/share/classes/javax/swing/plaf/basic/BasicComboPopup.java
+javax/swing/plaf/basic/BasicComboPopup.java
```

Comment:
I have created https://github.com/Itiviti/ikvm-openjdk6-b22 instead of storing them in this repo, please do the changes done there and raise a PR which we can merge there first.